e2e/tests/02_helm-detect-2.yaml

```
@@ -34,6 +34,23 @@ testcases:
     - result.code ShouldEqual 0
     - result.systemout ShouldContainSubstring "NAME                                       NAMESPACE   KIND         VERSION   REPLACEMENT   DEPRECATED   DEPRECATED IN   REMOVED   REMOVED IN"
     - result.systemout ShouldContainSubstring "kindled-toucan/kindled-toucan-basic-demo   demo1       Deployment   apps/v1                 false                        false"
+
+- name: helm2 detect in-cluster show-all wide namespaced confimaps
+  steps:
+  - script: pluto detect-helm --helm-version=2 --helm-store configmaps -A -t v1.15.0 -owide -n demo1
+    assertions:
+    - result.code ShouldEqual 0
+    - result.systemout ShouldContainSubstring "NAME                                       NAMESPACE   KIND         VERSION   REPLACEMENT   DEPRECATED   DEPRECATED IN   REMOVED   REMOVED IN"
+    - result.systemout ShouldContainSubstring "kindled-toucan/kindled-toucan-basic-demo   demo1       Deployment   apps/v1                 false                        false"
+
+- name: helm detect show all wide namespaced secrets
```

```suggestion
- name: helm2 detect show all wide namespaced secrets
```
nitpit: just to keep names consistent for our tests.